gradlew.bat

```diff
@@ -0,0 +1,90 @@
+@if "%DEBUG%" == "" @echo off
```

Comment:
Hey,

I checked and the gradle and android best practices say to include the
gradle wrapper at the top level of the project files.

It's ~impossible to test or debug gradle stuff without running --info and
--debug which are only available at the terminal level, and gradlew is the
recommended way to be doing CLI gradle.

-Clayton

On Wed, Sep 2, 2015 at 11:10 AM, Phillip Mates <notifications@github.com>
wrote:

> In gradlew.bat
> <https://github.com/dimagi/commcare-odk/pull/584#discussion_r38543432>:
>
> > @@ -0,0 +1,90 @@
> > +@if "%DEBUG%" == "" @echo off
>
> Do we need this or the gradlew file?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/dimagi/commcare-odk/pull/584/files#r38543432>.
>